frontend/server/handlers/healthz.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import * as path from 'path';
+import * as fs from 'fs';
+import fetch from 'node-fetch';
+import { Handler } from 'express';
+
+export interface IHealthzStats {
```

Comment:
Sorry, I think I forgot to identify this problem also in last PR.
This repo's convention is to never prefix interface with `I`, just use `HealthzStats` sounds good.

Reply:
fixed. renamed all the interfaces.

Reply:
renamed all interfaces.